git-open

```diff
@@ -93,14 +93,18 @@ else
   # custom GitLab
   gitlab_domain=$(git config --get gitopen.gitlab.domain)
   gitlab_port=$(git config --get gitopen.gitlab.port)
+  gitlab_protocol=$(git config --get gitopen.gitlab.protocol)
+  if [ -z "$gitlab_protocol" ]; then
+      gitlab_protocol=http
+  fi
   if [ -n "$gitlab_domain" ]; then
     if grep -q "$gitlab_domain" <<<$giturl; then
 
       # Handle GitLab's default SSH notation (like git@gitlab.domain.com:user/repo)
-      giturl=${giturl/git\@${gitlab_domain}\:/https://${gitlab_domain}/}
+      giturl=${giturl/git\@${gitlab_domain}\:/${gitlab_protocol}://${gitlab_domain}/}
 
       # handle SSH protocol (links like ssh://git@gitlab.domain.com/user/repo)
-      giturl=${giturl/#ssh\:\/\//https://}
+      giturl=${giturl/#ssh\:\/\//${gitlab_protocol}://}
```

Comment:
I didn't test this since my remote url use the following scheme: `git@gitlab.mydomain.com:mygroup/myrepository.git`